package-build.yml

```diff
@@ -15,14 +15,16 @@ spec:
           - ytt:
               paths:
               - config/package-bundle/config
-              - tmp/output-config.yml
+              - config/relase-config.yml
```

Comment:
release spelling is misspelled as relase

Reply:
Refactored